src/components/Search/loader.ts

```diff
@@ -1,15 +1,14 @@
-import { repoMap, frameworks, STUDENT_T } from '../../common/constants';
+import { repoMap, frameworks } from '../../common/constants';
 import { memoizedFetchRevisionForRepository } from '../../logic/treeherder';
 import { Changeset, Repository } from '../../types/state';
-import { Framework, TestVersion } from '../../types/types';
+import { Framework } from '../../types/types';
 
 const DEFAULT_VALUES = {
   newRev: null,
   newRevInfo: null,
   newRepo: 'try' as Repository['name'],
   frameworkId: 1 as Framework['id'],
   frameworkName: 'talos' as Framework['name'],
```

Comment:
Double checking, but Search/loader.ts should not have anything regarding testVersion. Removing any use of it on the file.